homeassistant/components/netgear/__init__.py

```diff
@@ -83,7 +83,9 @@ async def async_update_data():
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
-    unload_ok = await hass.config_entries.async_unload_platforms(entry, ["router"])
+    unload_ok = await hass.config_entries.async_unload_platforms(
+        entry, PLATFORMS_UNLOAD
```

Comment:
Wasn't the word `router` already a constant? Maybe just put that constant in a list?

Reply:
No because "router" is not a standard platform, that is because this integration uses a rather peculiar way of adding the entities due to new devices that can connect later on when the integartion is already setup.
This is done through signaling and with a callback in the router.py file here:
https://github.com/home-assistant/core/blob/2f605d51b32d6ef39b7313a6fa8bd7145ff9dc38/homeassistant/components/netgear/router.py#L78-L87

I noticed I got erros if I try to unload the sensor and device_tracker platforms, probably because the entities are added from the router.py file.

Reply:
What I meant here was `[KEY_ROUTER]` so you don't have to define a new line of code.

Reply:
Alright now I get it, although KEY_ROUTER is not per definition coupled to the router.py file. It is just a string used to store config data.

I like to have the PLATFORMS and PLATFORMS_UNLOAD next to eachother such that I can remember to think about it if I add something to PLATFROMS.

Reply:
I now use the KEY_ROUTER